cmd/experimental/kjobctl/pkg/cmd/list/list_interactive.go

```diff
@@ -0,0 +1,198 @@
+	interactiveExample = `  # List Interactive
+  kjobctl list interactive`
+)
+
+type InteractiveOptions struct {
+	Clock      clock.Clock
+	PrintFlags *genericclioptions.PrintFlags
+
+	Limit              int64
+	AllNamespaces      bool
+	Namespace          string
+	ProfileFilter      string
+	FieldSelector      string
+	LabelSelector      string
+	ClusterQueueFilter string
+
+	Client corev1.CoreV1Interface
+
+	genericiooptions.IOStreams
+}
+
+func NewInteractiveOptions(streams genericiooptions.IOStreams, clock clock.Clock) *InteractiveOptions {
+	return &InteractiveOptions{
+		PrintFlags: genericclioptions.NewPrintFlags("").WithTypeSetter(scheme.Scheme),
+		IOStreams:  streams,
+		Clock:      clock,
+	}
+}
+
+func NewInteractiveCmd(clientGetter util.ClientGetter, streams genericiooptions.IOStreams, clock clock.Clock) *cobra.Command {
+	o := NewInteractiveOptions(streams, clock)
+
+	cmd := &cobra.Command{
+		Use:                   "interactive [--selector key1=value1] [--field-selector key1=value1] [--all-namespaces]",
+		DisableFlagsInUseLine: true,
+		Short:                 "List Interactive",
+		Example:               interactiveExample,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			cmd.SilenceUsage = true
+			err := o.Complete(clientGetter)
+			if err != nil {
+				return err
+			}
+			return o.Run(cmd.Context())
+		},
+	}
+
+	o.PrintFlags.AddFlags(cmd)
+
+	util.AddAllNamespacesFlagVar(cmd, &o.AllNamespaces)
+	util.AddFieldSelectorFlagVar(cmd, &o.FieldSelector)
+	util.AddLabelSelectorFlagVar(cmd, &o.LabelSelector)
+	util.AddProfileFlagVar(cmd, &o.ProfileFilter)
+
+	return cmd
+}
+
+// Complete completes all the required options
+func (o *InteractiveOptions) Complete(clientGetter util.ClientGetter) error {
+	var err error
+
+	o.Limit, err = listRequestLimit()
+	if err != nil {
+		return err
+	}
+
+	o.Namespace, _, err = clientGetter.ToRawKubeConfigLoader().Namespace()
+	if err != nil {
+		return err
+	}
+
+	clientset, err := clientGetter.K8sClientset()
+	if err != nil {
+		return err
+	}
+
+	o.Client = clientset.CoreV1()
+
+	return nil
+}
+
+func (o *InteractiveOptions) ToPrinter(headers bool) (printers.ResourcePrinterFunc, error) {
+	if !o.PrintFlags.OutputFlagSpecified() {
+		printer := newInteractiveTablePrinter().
+			WithNamespace(o.AllNamespaces).
+			WithHeaders(headers).
+			WithClock(o.Clock)
+		return printer.PrintObj, nil
+	}
+
+	printer, err := o.PrintFlags.ToPrinter()
+	if err != nil {
+		return nil, err
+	}
+
+	return printer.PrintObj, nil
+}
+
+// Run performs the list operation.
+func (o *InteractiveOptions) Run(ctx context.Context) error {
+	var totalCount int
+
+	namespace := o.Namespace
+	if o.AllNamespaces {
+		namespace = ""
+	}
+
+	opts := metav1.ListOptions{
+		LabelSelector: constants.ProfileLabel,
+		FieldSelector: o.FieldSelector,
+		Limit:         o.Limit,
+	}
+
+	if len(o.ProfileFilter) > 0 {
+		opts.LabelSelector = fmt.Sprintf("%s,%s=%s", opts.LabelSelector, constants.ProfileLabel, o.ProfileFilter)
+	}
```

Comment:
```suggestion
	opts := metav1.ListOptions{
		LabelSelector: constants.ProfileLabel,
		Limit:         o.Limit,
	}

	if len(o.ProfileFilter) > 0 {
		opts.LabelSelector = fmt.Sprintf("%s=%s", constants.ProfileLabel, o.ProfileFilter)
	} else {
		opts.LabelSelector = constants.ProfileLabel
	}
```

nit: It's easier to follow and it makes more sense in my opinion.